ibl_to_nwb/brainwide_map/convert_brainwide_map_processed_only_local_testing.py

```diff
@@ -56,18 +56,20 @@
 # sessions_to_run = list(set(brain_wide_sessions) - set(already_written_processed_sessions))
 
 nwbfile_path = base_path / "nwbfiles" / session / f"{session}.nwb"
-nwbfile_path.parent.mkdir(exist_ok=True)
+os.makedirs(nwbfile_path.parent, exist_ok=True)
```

Comment:
```suggestion
nwbfile_path.parent.mkdir(exist_ok=True)
```

Why change this? Pathlib is generally more convenient

Reply:
Can add `parents=True` if that is necessary